java/src/main/java/com/google/cloud/dataproc/templates/gcs/README.md

```diff
@@ -34,3 +34,23 @@ bin/start.sh \
 --templateProperty gcs.spanner.output.saveMode=<Append|Overwrite|ErrorIfExists|Ignore> \
 --templateProperty gcs.spanner.output.primaryKey=<column[(,column)*] - primary key columns needed when creating the table>
 ```
+
+
+## 3. GCS to JDBC
+
+```
+export JARS=<gcs location for jdbc connector jar>
+
+GCP_PROJECT=<gcp-project-id> \
+REGION=<region>  \
+GCS_STAGING_LOCATION=<gcs-staging-bucket-folder> \
+bin/start.sh \
+-- --template GCSTOJDBC \
+--templateProperty project.id=<gcp-project-id> \
+--templateProperty gcs.jdbc.input.format=<csv | avro | parquet | orc> \
```

Comment:
have you tested it for it to works with "csv" ??
Since CSV does not have schema, generally we are not advising it when writing to databases.

Reply:
Although I tested it with csv and it is working fine but in that case it is taking all the column as string type. So, I guess better is to remove that option. and I did the same.